api/db/db.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+
+mongoose.connect('mongodb+srv://huntinator_server:UYjTXTeo6xyxrGUV@huntinator-crnnx.mongodb.net/test?retryWrites=true&w=majority', { useNewUrlParser: true }, err => {
+    if(err) console.log(err);
+});
+
+console.log('Connected to the Mongodb database');
```

Comment:
If you want this to actually log when the db connects then use the promise version of mongoose.connect():
```javascript
mongoose.connect(uri, options).then(
  () => { /** Put that log inside here */ },
  err => { /** handle connection error */ }
);
```
docs: https://mongoosejs.com/docs/connections.html#callback

Reply:
I was thinking about how I would have a success condition but I didn't know you could do it like this - thanks